parts/linux/cloud-init/artifacts/cse_main.sh

```diff
@@ -93,7 +93,9 @@ installContainerRuntime
 installTeleportdPlugin
 {{- end}}
 
+{{- if not IsByoCNI }}
```

Comment:
do you still want the vanilla cni plugins (installCNI inside installAzureCNI) even with byo? or does calico do everything fine itself? 

I assume the latter but just checking

Reply:
Yep the latter, Calico is all good without any CNI installed since their daemonset installs it. There is the interesting case of when we use it for just policies (ontop of azure CNI), but that will be configured in addon-manager.